ip/index.ts

```diff
@@ -630,20 +670,13 @@ function findIP(request: RequestLike, headers: Headers): string {
     return fastlyClientIP;
   }
 
-  // Akamai and Cloudflare
+  // Akamai
   // True-Client-IP
   const trueClientIP = headers.get("true-client-ip");
```

Comment:
Should we put this within the conditional that checks whether we're running on CF?

Reply:
No, because at https://developers.cloudflare.com/fundamentals/reference/http-request-headers/#true-client-ip-enterprise-plan-only Cloudflare states that it is for enterprise customers only and 
>There is no difference between the True-Client-IP and CF-Connecting-IP headers besides the name of the header. Some Enterprise customers with legacy devices need True-Client-IP to avoid updating firewalls or load-balancers to read a custom header name.

Which means the we'll find CF-Connecting-IP first so this is only to support Akamai, which doesn't have an env var we can look at.